### DISCUSSION TITLE
How to revoke rejections of studies

### DISCUSSION THREAD

**Discussion by DeeWongXB** [dcm4che/dcm4chee-arc-light#4304] [Q&A] [answered]
For test, I clicked **Reject matching studies** in **More functions** on the UI, which selected **Rentention expires**, after deleting those studies by **Permanent delete** in **More functions**, I want upload those data to dcm4chee-arc again, but I found once I finished uploading, those studies would always belong to **IOCM_EXPIRED**.
What can I do to move those rejected studies back to **DCM4CHEE**?

**Comment by vrindanayak** [comment, marked as answer]
Use `Restore Study` function / icon to restore the each of the studies you want to see in DCM4CHEE again from the IOCM_EXPIRED view.
